.github/workflows/push-dockerhub-readme.yml

```
@@ -0,0 +1,23 @@
+name: push readme to dockerhub
+
+on:
+  push:
+    paths:
+      - Docker/README.md
+    branches:
+      - edge
+  workflow_dispatch:
+
+jobs:
+  push-docker-readme:
```

```suggestion
  push-docker-readme:
    if: github.repository_owner == 'FreshRSS'
```

https://github.com/orgs/community/discussions/26409#discussioncomment-3251822
I do not know whether that condition can be put higher up, in the `on:` section

---

also implementet with the other push logic